data/json/itemgroups/Weapons_Mods_Ammo/guns.json

```diff
@@ -435,6 +435,7 @@
       { "item": "aksemi", "prob": 2, "charges-min": 0, "charges-max": 30 },
       { "item": "mk47", "prob": 1, "charges-min": 0, "charges-max": 30 },
       { "item": "m1carbine", "prob": 1, "charges-min": 0, "charges-max": 15 },
+      { "item": "m4_survival_rifle", "prob": 2, "charges-min": 0, "charges-max": 4 },
```

Comment:
I did a quick  comparison to the m1 carbine on gunbroker completed listings. I got 2, two, for the m4 survival rifle, and 602 hits for the m1 carbine.

Becuase the m4 survival rifle is chambered in .22 Hornet, and because it's a rare, expensive NFA item, I think that this item doesn't belong in 'common worn', but should be stuck in safes instead.

And it should be 1/301 times less common than the m1carbine.